dist-build/android-build.sh

```diff
@@ -71,9 +78,11 @@ if [ "$NDK_PLATFORM" != "$NDK_PLATFORM_COMPAT" ]; then
     ${LIBSODIUM_ENABLE_MINIMAL_FLAG} \
     --host="${HOST_COMPILER}" \
     --prefix="${PREFIX}" \
+    --includedir="${INCLUDE_PATH}" \
+    --libdir="${LIB_PATH}" \
     --with-sysroot="${TOOLCHAIN_DIR}/sysroot" || exit 1
 
-  grep -E '^#define ' config.log | sort -u >config-def.log
+  egrep '^#define ' config.log | sort -u >config-def.log
```

Comment:
`grep -E` is more portable.

Reply:
sorry again I didn't actually change this line another lost change from stable.